build/Linux+BSD/portable/Recipe

```diff
@@ -74,9 +74,23 @@ cd AppImageKit
 cd ..
 
 # MuseScore's dependencies:
+
+# get CMAKE 3.5 (Centos 6 only has CMAKE 2.8.12.2)
+CMAKE_URL="http://www.cmake.org/files/v3.5/cmake-3.5.1-Linux-x86_64.tar.gz"
+mkdir cmake && wget --no-check-certificate --quiet -O - ${CMAKE_URL} | tar --strip-components=1 -xz -C cmake
+export PATH=${PWD}/cmake/bin:${PATH}
+
+# get Qt5.6 (Centos 6 doesn't have QtWebEngine)
+mkdir qt5 && wget -q -O qt5.zip http://utils.musescore.org.s3.amazonaws.com/qt560.zip
```

Comment:
An alternative is to download the official package and script it see https://github.com/benlau/qtci